subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -2104,10 +2104,13 @@ static void *radio_ccm_ext_rx_pkt_set(struct ccm *cnf, uint8_t phy, uint8_t pdu_
 	NRF_CCM->ENABLE = CCM_ENABLE_ENABLE_Disabled;
 	NRF_CCM->ENABLE = CCM_ENABLE_ENABLE_Enabled;
 
-	mode = (CCM_MODE_MODE_Decryption << CCM_MODE_MODE_Pos) &
+#if defined(CONFIG_SOC_COMPATIBLE_NRF54LX)
+	/* NOTE: Use fast decryption as rx data is decrypt after payload is received, compared to
+	 *       decrypting in parallel with radio reception of address in nRF51/nRF52/nRF53.
+	 */
+	mode = (CCM_MODE_MODE_FastDecryption << CCM_MODE_MODE_Pos) &
```

Comment:
`CCM_MODE_MODE_FastDecryption` only exists for `CONFIG_SOC_COMPATIBLE_NRF54LX`?

Reply:
yes and later series